pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess we still do not want the byte order mark. Could you please revert this file?

Reply:
again ?